src/Rebing/GraphQL/Support/PaginationType.php

```diff
@@ -59,6 +59,18 @@ protected function getPaginationFields($typeName)
                 'resolve'       => function(LengthAwarePaginator $data) { return $data->lastItem(); },
                 'selectable'    => false,
             ],
+            'last_page' => [
+                'type'          => GraphQLType::int(),
```

Comment:
This would be wrapped in `GraphQLType::nonNull()` as the the returned value of `lastPage()` is always defined as int

Reply:
Ok